core/storage/base_model/gae_models.py

```diff
@@ -1507,12 +1507,12 @@ def get_version_string(self):
     @classmethod
     def export_data(cls, user_id):
         # type: (Text) -> Dict[Text, Dict[Text, Text]]
-        metadata_models = (
+        results = (
             cls.query(cls.committer_id == user_id).fetch(
                 projection=[cls.commit_type, cls.commit_message]))
 
         metadata_models = cast(
-            List[BaseSnapshotMetadataModel], metadata_models)
+            List[BaseSnapshotMetadataModel], results)
```

Comment:
Same note as above, here and elsewhere. You can break after the comma if you want the actual "definition" of metadata_models to be on its own line, that could make it more readable.

Reply:
Done